lib/kitchen/driver/azurerm.rb

```diff
@@ -224,8 +224,20 @@ def create(state)
           vaultResourceGroup: config[:vault_resource_group],
         }
 
+        # Set a randomly generated password or one from the environment if an SSH key isn't present
```

Comment:
This isn't always accurate; `config[:password]` may be provided by the user rather than randomly generated